### ISSUE TITLE
[$1000] Web- Admin - Nothing appears if workspace is named <workspace_name starting with <

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#20080]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to staging expensify on web chrome
2. Create a new workspace > Go to admins channel
3. Change the name of the workspace to < followed by some word
4. Result: See that in the admins room , nothing is displayed after 'to' at the end

## Expected Result:
The updated workspace name should appear properly even when it is named as < followed by any word

## Actual Result:
Nothing appears if workspace is named <workspace_name starting with <. Also the whole line is shown in a different color.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.22.0

**Reproducible in staging?:** yes

**Reproducible in production?:** yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/ab76c685-ca7c-436d-893b-3073143c2bb7

https://github.com/Expensify/App/assets/93399543/7b6e091d-86e8-40b5-8f0a-0af3d68c5216

**Expensify/Expensify Issue URL:**

**Issue reported by:** @avi-shak-jha

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1684769611423049

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014aa9edd793117534</li>
        <li>Upwork Job ID: 1666602306347040768</li>
        <li>Last Price Increase: 2023-12-20</li>
    </ul>
</details>

**Comment by BhuvaneshPatil** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Web- Admin - Nothing appears if workspace is named <workspace_name starting with <

### What is the root cause of that problem?
The root cause for this problem is that, the HTML coming from backend is not encoded.

![image](https://github.com/Expensify/App/assets/27822551/7d837b34-aef1-4a18-b7a8-fefc8843a3e3)

The html should have `&lt;` instead of `<`


### What changes do you think we should make in order to solve the problem?

1. The one possible solution is to encode the html on backend and send that, when we change the name and API is called. 
2. To pass a new prop `shouldEncodeHtml` to `ReportActionItemFragment` component that will decide if we will encode the html before passing it to `<RenderHTML/>` component.
The value for this prop will be decided in `ReportActionItemMessage` component, based on `actionName`
```
const shouldEncodeHtml = props.action.actionName === CONST.RPORT.ACTIONS.TYPE.POLICYCHANGELOG.UPDATE_NAME;.
```
This needs to be worked upon and I am looking for more options

### What alternative solutions did you explore? (Optional)

**Comment by arielgreen** [comment]
This is okay to go external

**Comment by MonilBhavsar** [comment]
Similar root cause as https://github.com/Expensify/App/issues/17758

**Comment by mollfpr** [comment]
Agree with @MonilBhavsar. We can hold this, and I'll put this issue in to the test case on https://github.com/Expensify/App/pull/20239

**Comment by arielgreen** [comment]
Reassigning, team change

**Comment by alexpensify** [comment]
Got it, let's keep this party going!

**Comment by alexpensify** [comment]
Still accepting proposals here

**Comment by alexpensify** [comment]
Still on hold

**Comment by alexpensify** [comment]
@mollfpr and @MonilBhavsar - should we move this one to weekly since it's on Hold?

**Comment by alexpensify** [comment]
For my notes, I think we are on hold because of this PR: 

https://github.com/Expensify/App/pull/20239

**Comment by alexpensify** [comment]
Still on hold

**Comment by alexpensify** [comment]
Still on hold and waiting on an update if we should move to Weekly

**Comment by alexpensify** [comment]
Reassigning another `bug` team member, I'm going OOO until Monday, July 10, and will take it back then if it's not closed by then. 

Required action from the 🐛 team: Check to see if this one goes back to a daily and we can resume work here.

**Comment by dylanexpensify** [comment]
reviewing today and posting next steps!

**Comment by dylanexpensify** [comment]
On hold it seems!

**Comment by dylanexpensify** [comment]
Hold, @MonilBhavsar shall we make this weekly?

**Comment by MonilBhavsar** [comment]
Yes, let's make it weekly

**Comment by alexpensify** [comment]
Alright, removing the other 🐛 team member. I'm back online and see that this is our current state: 

Still on hold

**Comment by alexpensify** [comment]
@MonilBhavsar are we still on hold here: https://github.com/Expensify/App/pull/20239

**Comment by MonilBhavsar** [comment]
Yes, sorry, It's taking some time

**Comment by alexpensify** [comment]
All good, I've update this one to be Weekly again.

**Comment by alexpensify** [comment]
Weekly Update: On hold

**Comment by alexpensify** [comment]
Weekly Update: On hold

**Comment by alexpensify** [comment]
Weekly Update: On hold

**Comment by alexpensify** [comment]
Weekly update: On hold

**Comment by alexpensify** [comment]
Weekly update: On hold

**Comment by alexpensify** [comment]
Weekly update: On hold

**Comment by alexpensify** [comment]
Weekly update: On hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold but looks like the PR might get moving again

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly update: On Hold

**Comment by alexpensify** [comment]
Weekly Update: @MonilBhavsar any news here for my last question?

**Comment by alexpensify** [comment]
Weekly update: Waiting for a reply, I've sent a message in chat to try to figure out if we should close this one.

**Comment by MonilBhavsar** [comment]
This issue is most probably fixed. We can retest and close

**Comment by alexpensify** [comment]
Weekly Update: I need to retest to confirm the next steps.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by alexpensify** [comment]
Ok, there we go, we can close this out because it's not appearing in the tests.
